lib/cretonne/meta/gen_instr.py

```diff
@@ -269,7 +273,13 @@ def gen_opcodes(groups, fmt):
                                 'Type inferred from {}.'
                                 .format(i.ins[i.format.typevar_operand]))
                 # Enum variant itself.
-                fmt.line(i.camel_name + ',')
+                if is_first_opcode:
+                    fmt.doc_comment('We explicitly set this to 1 to allow the NonZero optimization,')
+                    fmt.doc_comment('meaning that SIZEOF(Option<Opcode>) == SIZEOF(Opcode)')
```

Comment:
It looks like `flake8` is complaining that these two lines are longer than 79 characters. That's why the Travis build is failing.

I actually think it would be OK to just delete them (or change them into a Python comment) since you already explain what's going on in the block comment above. The non-zero details don't need to go into the opcode documentation we're generating here.